backend/src/impl/benchmark_configs/globalbench.json

```diff
@@ -37,7 +37,15 @@
                 {"op": "max", "group_by": ["source_language"], "skip_group_system": false},
                 {"op": "weighted_sum", "weight": "source_language", "weight_map": "ling_weight", "skip_group_system": false}
             ]
+        }, 
+        {
+            "name": "Languages",
```

Comment:
1. could you make the name more descriptive, "most underserved languages"?
2. could we also add the demographic-weighted value, where you multiply by `POP_WEIGHT` at the end?